react/src/components/KeypairResourcePolicySettingModal.tsx

```diff
@@ -324,72 +294,59 @@ const KeypairResourcePolicySettingModal: React.FC<
             <Row gutter={16}>
               <Col span={12}>
                 <FormItemWithUnlimited
-                  form={form}
-                  fieldName={['parsedTotalResourceSlots', 'cpu']}
-                  checkboxFieldName="totalResourceSlotsCPUUnlimited"
+                  name={['parsedTotalResourceSlots', 'cpu']}
+                  unlimitedValue={undefined}
                   label={resourceSlotsDetails?.cpu.description}
-                  fallbackValue={parsedTotalResourceSlots?.cpu}
-                  render={(disabled) => (
+                  children={
                     <InputNumber
                       min={0}
                       max={512}
-                      disabled={disabled}
                       addonAfter={resourceSlotsDetails?.cpu.display_unit}
                     />
-                  )}
+                  }
                 />
               </Col>
               <Col span={12}>
                 <FormItemWithUnlimited
-                  form={form}
-                  fieldName={['parsedTotalResourceSlots', 'mem']}
-                  checkboxFieldName="totalResourceSlotsMEMUnlimited"
+                  name={['parsedTotalResourceSlots', 'mem']}
+                  unlimitedValue={undefined}
                   label={resourceSlotsDetails?.mem.description}
-                  fallbackValue={parsedTotalResourceSlots?.mem}
-                  render={(disabled) => (
-                    <DynamicUnitInputNumber disabled={disabled} />
-                  )}
+                  children={<DynamicUnitInputNumber />}
                 />
               </Col>
             </Row>
             <Row gutter={16} align="bottom">
               <Col span={12}>
                 <FormItemWithUnlimited
-                  form={form}
-                  fieldName={['parsedTotalResourceSlots', 'cuda.device']}
-                  checkboxFieldName="totalResourceSlotsGPUUnlimited"
+                  name={['parsedTotalResourceSlots', 'cuda.device']}
+                  unlimitedValue={undefined}
                   label={resourceSlotsDetails?.['cuda.device'].description}
-                  fallbackValue={parsedTotalResourceSlots?.['cuda.device']}
-                  render={(disabled) => (
+                  children={
                     <InputNumber
                       min={0}
                       max={64}
-                      disabled={disabled}
                       addonAfter={
                         resourceSlotsDetails?.['cuda.device'].display_unit
                       }
                     />
-                  )}
+                  }
                 />
               </Col>
```

Comment:
You can set the `children` between open tag and close tag.
```suggestion
                <FormItemWithUnlimited
                  name={['parsedTotalResourceSlots', 'cuda.device']}
                  unlimitedValue={undefined}
                  label={resourceSlotsDetails?.['cuda.device'].description}
                >
                  <InputNumber
                      min={0}
                      max={64}
                      addonAfter={
                        resourceSlotsDetails?.['cuda.device'].display_unit
                      }
                    />
                </FormItemWithUnlimited>
              </Col>
```